testing/asserts.ts

```diff
@@ -411,6 +411,43 @@ export function assertNotStrictEquals(
   );
 }
 
+/**
+ * Make an assertion that `actual` and `expected` are almost equal numbers through
+ * a given epsilon. It can be used to take into account IEEE-754 double-precision
```

Comment:
Not a change request, but I'm little unsure if this `epsilon` usage is enough descriptive to everyone. `difference` might be easier to understand, I guess

Reply:
Good point

Reply:
Another common name for this parameter is `delta`. But `difference` is the most likely of these three to be in the vocabulary of the average user.

Reply:
`tolerance` may be a good option

Reply:
I'll change it to `tolerance`, seems to be more close to the meaning of `epsilon` I guess

Reply:
@lowlighter Thanks for updating!